src/frame/worker/crypto/recrypt/tests/RecryptWasm.test.ts

```diff
@@ -52,6 +52,48 @@ describe("RecryptWasm", () => {
         });
     });
 
+    describe("rotateGroupPrivateKeyWithRetry", () => {
+        const groupPrivateKey = new Uint8Array([22, 33, 44]);
+        it("should result in an error when mocked generateKeyPair returns Uint8Array of zeros for new privateKey", () => {
+            jest.spyOn(Recrypt.getApi(), "generatePlaintext");
+            Recrypt.rotateGroupPrivateKeyWithRetry(groupPrivateKey).engage(
+                (error) => {
+                    expect(error.message).toEqual("Key rotation failed.");
```

Comment:
I'd just remove this line. There's little to no benefit in having to update this test if we decide to change the text of the error message. The assert below this is sufficient.

Reply:
done